internal/security/authz/authz.go

```diff
@@ -166,21 +166,31 @@ func Get(t testing.TB, dut *ondatra.DUTDevice) (*authzpb.GetResponse, *Authoriza
 		t.Fatalf("Could not connect gnsi %v", err)
 	}
 	resp, err := gnsiC.Authz().Get(context.Background(), &authzpb.GetRequest{})
+	nopolicyexist := 0
 	if err != nil {
-		t.Fatalf("Authz.Get request is failed on device %s: %v", dut.Name(), err)
-	}
-
-	t.Logf("Authz.Get response is %s", prettyPrint(resp))
-	if resp.GetVersion() == "" {
-		t.Errorf("Version is not set in Authz.Get response")
-	}
-	if resp.GetCreatedOn() > uint64(time.Now().UnixMicro()) {
-		t.Errorf("CreatedOn value can not be larger than current time")
+		statusError, _ := status.FromError(err)
```

Comment:
looks like you are throwing away an err, why?
(probably a comment like; "ignoring error here for <reason>" is good)

Reply:
We are not throwing away error, when we have empty get (no existing policy on device scenario) we can not have this check so we added "nopolicy" flag to verify if the device has existing policy or not. Incase if the device has no policy during test start we can not compare any value as part of Get output (which emits FAILED PRECONDITION ERROR). So the code from 177 had to be excluded for No policy Get scenario. However the code will get exercised incase the device already has Authz policy installed.

Reply:
Sorry I think you are throwing away the error:

  statusError, _ := status.FromError(err)
  
  you are throwing away the error from the 'status.FromError()'.
  Admittedly that PROBABLY Shouldn't error, but it's bad form to not check the return there.

Reply:
Got it. Added check if FromError is able to identify mapped GRPC status error message if it returns unknown then flagged fatal with Error message.